src/utils/decorators/input/input.scss

```diff
@@ -195,7 +195,7 @@
   height: 15px;
   position: absolute;
   right: 5px;
-  top: 6px;
+  top: 28px;
```

Comment:
It restores the value to what it was before. We want to release this so won't be re-writing the code more than necessary